src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java

```diff
@@ -59,6 +59,7 @@
   private static final String DEV_APPSERVER_PY = "bin/dev_appserver.py";
   private static final String JAVA_APPENGINE_SDK_PATH =
       "platform/google_appengine/google/appengine/tools/java/lib";
+  private static final String JAVA_COMPONENTS_PATH = "platform/gcd/.appengine";
```

Comment:
So again, this is just checking for the wrong thing.  That directory is for the legacy datastore emulator which has nothing to do with Java or any Java components.

How far are we from having the requiring staging code as separate artifacts on maven central?  I don't think it's appropriate for the plugin to be grabbing individual jars from the SDK as this is not a public interface.